databus-api/src/test/java/com/bazaarvoice/emodb/databus/api/SubscriptionJsonTest.java

```diff
@@ -63,7 +63,7 @@ public void testSubscriptionJsonWithEventTtlGreaterThan365Days() throws Exceptio
         JsonHelper.fromJson(subscriptionJsonString, Subscription.class);
     }
 
-    @Test(expectedExceptions=IllegalArgumentException.class, expectedExceptionsMessageRegExp=".* EventTtl must be >0 .*")
+    @Test(expectedExceptions=IllegalArgumentException.class, expectedExceptionsMessageRegExp=".* EventTtl must be >0(?s).*")
     public void testSubscriptionJsonWithNegativeEventTtl() throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.Exception', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)